packages/strapi-utils/lib/sanitize-entity.js

```diff
@@ -12,7 +12,13 @@ const {
 const { ID_ATTRIBUTE } = constants;
 
 const sanitizeEntity = (dataSource, options) => {
-  const { model, withPrivate = false, isOutput = true, includeFields = null } = options;
+  const {
+    model,
+    withHidden = true,
```

Comment:
Won't this be kept in the CM response then?

Reply:
Nope, in the CM's sanitize we're adding a `withHidden = false`. This allow us to not make any changes in the default sanitize's behavior and only prevent hidden fields to appear in the CM inputs/responses

Reply:
Good then